mold.h

```diff
@@ -311,7 +311,7 @@ inline u64 read_uleb(u8 const*&buf) {
 }
 
 inline i64 uleb_size(u64 val) {
-#pragma unroll
+#pragma GCC unroll 8
```

Comment:
This is a valid pragma for Clang.

https://clang.llvm.org/docs/AttributeReference.html#pragma-unroll-pragma-nounroll

Reply:
`#pragma GCC unroll <n>` is supported by both GCC and Clang, whereas `#pragma unroll` is only supported by Clang. Using the GCC-compatible notation fixes an *unknown pragma* warning when compiling with GCC.